php/stats.php

```diff
@@ -74,7 +74,7 @@
 }
 
 // unique airlines, unique planes, total distance (mi), average distance (localized), average duration
```

Comment:
Comment presumably wants updating to match...

Reply:
And because GitHub CR is daft, I can't leave this comment at line 84 below... But presumably this doesn't actually result in anything being output, on the website, it just makes it available to JS, so would also need a change to https://github.com/jpatokal/openflights/blob/89643b0b3a6d750459c40799b45901beb080d805/openflights.js#L1558-L1564 too?

Reply:
I'll make the necessary changes and submit another pull. Thanks @reedy